src/appforms/src/core/040-Model06Submission.js

```diff
@@ -379,6 +386,34 @@ appForm.models = function(module) {
 
     this.setFormFields(formFields);
   };
+
+  /**
+   * Submission.prototype.pruneRemovedFields - Pruning fields that have been deleted or removed from the form
+   *
+   * @return {type}  description
+   */
+  Submission.prototype.pruneRemovedFields = function(cb) {
+    var that = this;
+    var formFields = this.getFormFields();
+    var newFields = [];
+
+    this.getForm(function(err, form) {
+      if (err) {
+        return cb(err);
+      }
+      //Loop and push matching fields
+      _.each(formFields, function(field) {
+        var fieldId = field.fieldId;
+        if (form.fields.hasOwnProperty(fieldId)) {
+          newFields.push(field);
```

Comment:
There will need to be code to delete any files associated with the form field if it is a file field type (photo/signature). Otherwise, there could be a lot of stale files.

See https://github.com/feedhenry/fh-js-sdk/blob/master/src/appforms/src/core/040-Model06Submission.js#L786 for an example where files are removed for an input value.